apps/mymedicare_cb/views.py

```diff
@@ -32,6 +33,10 @@
 
 # For SLS auth workflow info, see apps/mymedicare_db/README.md
 def authenticate(request):
+    # Create authorization flow trace UUID, if not existing from dispatch()
+    if request.session.get('auth_uuid', None) is None:
+        request.session['auth_uuid'] = str(uuid.uuid4())
+
```

Comment:
Added this check to handle a case when using MSLS testing locally. It is probably not needed otherwise.

Reply:
LGTM